includes/Modules/Analytics.php

```diff
@@ -1420,7 +1389,7 @@ protected function validate_report_metrics( $metrics ) {
 	 *
 	 * @param array $dimensions The dimensions to validate.
```

Comment:
While this works as-is, let's update the type to expect an array of the appropriate instance.
```suggestion
	 * @param Google_Service_AnalyticsReporting_Dimension[] $dimensions The dimensions to validate.
```